### PR TITLE
fix(react-kit): prevent to open DateInput popup on click if DateInput is disabled

### PR DESCRIPTION
**Pull request by green176**
prevent to open DateInput popup on click if DateInput is disabled

### CODE REVIEW

File: packages/react-kit/src/components/DateInput/DateInput.tsx [hunk 1]
```diff
@@ -433,6 +433,10 @@ class RawDateInput extends React.Component<TDateInputFullProps, TDateInputState>
 	};
 
 	onSteppableInputClick = (e: React.MouseEvent<HTMLElement>) => {
+		if (this.props.isDisabled) {
```
Comment (raveclassic) [inline packages/react-kit/src/components/DateInput/DateInput.tsx]: Why here and not in `SteppableInput`?

Comment (green176) [reply packages/react-kit/src/components/DateInput/DateInput.tsx]: Ok, I moved condition into SteppableInput

Comment (raveclassic) [reply packages/react-kit/src/components/DateInput/DateInput.tsx]: The same applies to `SteppableInput`

Comment (green176) [reply packages/react-kit/src/components/DateInput/DateInput.tsx]: Ok! Now I moved it to Input component

**Pull request merged**